src/imageuploadengine.js

```diff
@@ -85,19 +72,6 @@ export default class ImageUploadEngine extends Plugin {
 				}
 			}
 		} );
-
-		// Model to view converter for image's `uploadId` attribute.
-		editor.editing.modelToView.on( 'addAttribute:uploadId:image', ( evt, data, consumable ) => {
-			if ( !consumable.consume( data.item, eventNameToConsumableType( evt.name ) ) ) {
-				return;
-			}
-
-			const modelImage = data.item;
-			const viewFigure = editor.editing.mapper.toViewElement( modelImage );
-			const viewImg = viewFigure.getChild( 0 );
-
-			viewImg.setAttribute( 'src', this.placeholder );
-		} );
 	}
```

Comment:
Engine part will work stand-alone without this converter. Now all upload UI updates can be done using `uploadStatus` attribute as in `ImageUploadProgress`.